modules/contentigniteBidAdapter.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+import * as utils from 'src/utils'
+import { config } from 'src/config'
+import { registerBidder } from 'src/adapters/bidderFactory'
+
+const BIDDER_CODE = 'contentignite'
+
+export const spec = {
+  code: BIDDER_CODE,
+  pageID: Math.floor(Math.random() * 10e6),
+
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.accountID && bid.params.zoneID)
+  },
+
+  buildRequests: function(validBidRequests) {
+    var i
+    var zoneID
+    var bidRequest
+    var accountID
+    var keyword
+    var requestURI
+    var serverRequests = []
+    var zoneCounters = {}
+
+    for (i = 0; i < validBidRequests.length; i++) {
+      bidRequest = validBidRequests[i]
+      zoneID = utils.getBidIdParameter('zoneID', bidRequest.params)
+      accountID = utils.getBidIdParameter('accountID', bidRequest.params)
+      keyword = utils.getBidIdParameter('keyword', bidRequest.params)
+
+      if (!(zoneID in zoneCounters)) {
+        zoneCounters[zoneID] = 0
+      }
+
+      requestURI =
+        location.protocol + '//serve.connectignite.com/adserve/;type=hbr;'
+      requestURI += 'ID=' + encodeURIComponent(accountID) + ';'
+      requestURI += 'setID=' + encodeURIComponent(zoneID) + ';'
+      requestURI += 'pid=' + encodeURIComponent(spec.pageID) + ';'
+      requestURI += 'place=' + encodeURIComponent(zoneCounters[zoneID]) + ';'
+
+      // append the keyword for targeting if one was passed in
+      if (keyword !== '') {
+        requestURI += 'kw=' + encodeURIComponent(keyword) + ';'
+      }
+
+      zoneCounters[zoneID]++
+      serverRequests.push({
+        method: 'GET',
+        url: requestURI,
+        data: {},
+        bidRequest: bidRequest
+      })
+    }
+    return serverRequests
+  },
+
+  interpretResponse: function(serverResponse, bidRequest) {
+    var bidObj = bidRequest.bidRequest
+    var bidResponses = []
+    var bidResponse = {}
+    var isCorrectSize = false
+    var isCorrectCPM = true
+    var CPM
```

Comment:
Does `CPM` need to be in all caps?  Can you make it consistent with the other variables syntax (camelCase)?

Reply:
Have reverted to lowercase `cpm`